### DISCUSSION TITLE
Can i change table with users?

### DISCUSSION THREAD

**Discussion by verylovestars** [strapi/strapi#7482] [Legacy Topics]
Can I change the table with users, in particular its name and model, as well as the password encryption method? i need to integrate a plugin with an existing user table

**Comment by verylovestars** [comment]
I did this by configuring a plugin responsible for managing user privileges.

**Reply by derrickmehaffy** [reply]
Can you give a little more context as to what you did in case anyone else searches and finds this discussion? Thank you! :hugs:
